app/jobs/delete_user.rb

```diff
@@ -0,0 +1,14 @@
+class DeleteUser
+  attr_reader :user
+
+  def initialize(user)
+    @user = user
+  end
+
+  def perform
+    profile = User.find(user['id'])
+    email = profile.email
+    profile.destroy
```

Comment:
Is it possible this returns false and account will not be destroyed, but the email will be sent anyway?